download_and_create_reference_datasets/v03/write_cached_reference_dataset_query_ht.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+import argparse
+
+import hail as hl
+
+from hail_scripts.reference_data.config import CONFIG
+
+from v03_pipeline.lib.misc.io import write
+from v03_pipeline.lib.model import CachedReferenceDatasetQuery, Env, ReferenceGenome
+from v03_pipeline.lib.paths import cached_reference_dataset_query_path
+
+
+def run(
+    env: Env,
+    reference_genome: ReferenceGenome,
+    query: CachedReferenceDatasetQuery,
+):
+    config = CONFIG[query.dataset][reference_genome.v02_value]
+    ht = (
+        config['custom_import'](config['source_path'], reference_genome.v02_value)
+        if 'custom_import' in config
+        else hl.read_table(config['path'])
+    )
+    ht = query.query(ht, env=env, reference_genome=reference_genome)
+    destination_path = cached_reference_dataset_query_path(
+        env,
+        reference_genome,
+        query,
+    )
+    print(f'Uploading ht to {destination_path}')
```

Comment:
good suggestion, we don't have a standardized logger in this repo (mostly because everything in here is super ad-hoc), but now seems like a good time for it.  I'm gonna make a ticket though and do that offline, since we have several other places to change, for example:

https://github.com/broadinstitute/seqr-loading-pipelines/blob/benb/gcnv_v3/download_and_create_reference_datasets/v03/write_combined_interval_reference_ht.py#L40

https://github.com/broadinstitute/seqr-loading-pipelines/blob/benb/gcnv_v3/download_and_create_reference_datasets/v03/write_combined_reference_ht.py#L47

Reply:
https://github.com/broadinstitute/seqr-loading-pipelines/issues/566